src/app/routes/routeProjects.ts

```diff
@@ -8,65 +8,100 @@ import { verifyAccessToken } from "../middlewares/verifyAccessToken";
 import { BUILD_MESSAGE } from "../../config/constants";
 import { ProjectService } from "../../domains/projectService";
 import { UserService } from "../../domains/userService";
-import { container } from "../../domains/@config/di.config";
+import { container } from "../../config/di.config";
```

Comment:
```suggestion
  framework: z.union(keys(CLIENT_FRAMEWORK_INFO).map(framwork => z.literal(framwork))),
```
이렇게는 어떨까요? (`keys`는 lodash 메서드입니다..!)